security/fma-contract-version-upgrade.md

```diff
@@ -0,0 +1,122 @@
+# [Project Name]: Failure Modes and Recovery Path Analysis
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Introduction](#introduction)
+- [Failure Modes and Recovery Paths](#failure-modes-and-recovery-paths)
+  - [Missing side effect on `.selector` access bug (Bug fixed after 0.8.15)](#missing-side-effect-on-selector-access-bug-bug-fixed-after-0815)
+  - [Storage write removal before conditional termination (Bug fixed after 0.8.15)](#storage-write-removal-before-conditional-termination-bug-fixed-after-0815)
+  - [Change in Default EVM version between different Solidity versions](#change-in-default-evm-version-between-different-solidity-versions)
+- [Audit Requirements](#audit-requirements)
+- [Action Items](#action-items)
+- [Appendix](#appendix)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+|                     |                                    |
+| ------------------- | ---------------------------------- |
+| Author              | Michael Amadi                      |
+| Created at          | 2024-08-20                         |
+| Needs Approval From | _Security Reviewer Name_           |
+| Other Reviewers     | _Reviewer Name 1, Reviewer Name 2_ |
```

Comment:
You can put needs approval from as Matt Solomon and put me (Mark Tyneway) as other reviewer